tests/vector/test_vector_env.py

```diff
@@ -53,16 +52,6 @@ def test_vector_env_equal(shared_memory):
     sync_env.close()
 
 
-def test_custom_space_vector_env():
```

Comment:
This was removed because trying to do `env = VectorEnv(...)` makes about as much sense as trying to do `env = gym.Env(...)`